meta/main.yml

```diff
@@ -0,0 +1,30 @@
+dependencies: []
+
+galaxy_info:
+  short_description: Grants Conjur machine identity
+  description: Grants Conjur machine identity to hosts
+  company: CyberArk
+  license: Apache
+  author:
+    - Dustin Collins (@dustinmm80)
+    - Oren Ben Meir (@orenbm)
+    - Jason Vanderhoof (@jvanderhoof)
+    - Kumbirai Tanekha (@doodlesbykumbi)
+    - Gary Moon (@garymoon)
+
+  min_ansible_version: '2.3'
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
+    - name: EL
+      versions:
+        - 7
+
+  galaxy_tags:
+    - identity
+    - cyberark
+    - conjur
+    - security
```

Comment:
Is this file going to override our current metadata for the module?

Reply:
How will the conjur_secret installation/use differ from this new identity role?

Reply:
According to the [documentation on roles](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#using-roles),
> If roles/x/meta/main.yml exists, any role dependencies listed therein will be added to the list of roles (1.3 and later).

The `meta` directory is for use by roles specifically, so the lookup plugin will be unaffected. Currently, "metadata" is found within the lookup plugin Python file itself, as per Ansible requirements.

Reply:
this file is at `meta/main.yml` and not at `roles/{conjur-host-identity}/meta/main.yml` - is it in the right location?